shared/rakefiles/tests/spec/vars_spec.rb

```diff
@@ -109,6 +143,16 @@ def scrub_env
     Vars.set_vars(env, project_type)
     expect(ENV).to have_received(:[]=).with("TF_VAR_nonce", a_value)
   end
+
+  it "Create directory path" do
```

Comment:
Delete.

Reply:
sorry @mrtyler I can not understand this.

Reply:
Deleted, I used this function in the implementation of a solution that didn't work as expected.